sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -402,4 +384,30 @@ export class ApplicationController extends BaseController {
       formName: applicationProgramYear.programYear.formName,
     } as ProgramYearOfApplicationDto;
   }
+
+  /**
+   * API to fetch application details by applicationId and studentId.
+   * This API will be used by ministry users.
+   * @param applicationId
+   * @param userId
+   * @returns Application details
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @Get("aest/:applicationId/:studentId")
+  async getByStudentAndApplicationId(
+    @Param("applicationId") applicationId: number,
+    @Param("studentId") studentId: number,
+  ): Promise<GetApplicationDataDto> {
+    const application = await this.applicationService.getApplicationByIdAndUser(
+      applicationId,
+      0,
```

Comment:
It seems that the method could be called with or without the user id, which makes the userId an optional parameter.
Can you please change the method to _userId_ be an optional parameter?